app-ios/App/DroidKaigi2023/DroidKaigi2023/App.swift

```diff
@@ -1,8 +1,11 @@
 import Navigation
 import SwiftUI
+import Assets
```

Comment:
Please sort import orders 🙏🏼 
I think SwiftLint warns this line when build iOS App.